src/spikeinterface/core/generate.py

```diff
@@ -845,6 +845,85 @@ def clean_refractory_period(times, refractory_period):
     return times
 
 
+def _add_spikes_to_spiketrain(
+    spike_indices,
+    spike_labels,
+    segment_indices=[],
```

Comment:
```suggestion
    segment_indices=None,
```
See https://github.com/SpikeInterface/spikeinterface/pull/2345

Reply:
To quote Joe: "Thanks I did not know about that, I am learning so much working on the SI codebase 😍!"